resources/language/messages.pot

```diff
@@ -244,7 +244,7 @@ msgid "Use proxy settings"
 msgstr ""
 
 msgctxt "#32107"
-msgid "Proxy from Elementum settings"
+msgid "From Elementum settings"
```

Comment:
@galeksandrp 
since you changed template - you also need to change all translated files, otherwise the will lose old translation after running synchronization scripts in `scripts` dir.

`for file in resources/language/*/*.po; do sed -i 's/msgid "Proxy from Elementum settings"/msgid "From Elementum settings"/' "$file"; done`
will do the trick.

Reply:
Done, thanks!